architecture/service_catalog/service_catalog_cli.adoc

```diff
@@ -0,0 +1,438 @@
+[[architecture-additional-concepts-service-catalog-cli]]
+= Service Catalog CLI
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[service-catalog-cli-overview]]
+== Overview
+The
+xref:../../architecture/service_catalog/index.adoc#service-catalog-design[basic
+workflow] of interacting with the service catalog is that:
+
+* The cluster administrator installs and registers a broker server to make
+available its services.
+* The users use those services by instantiating them in an OpenShift project and
+linking those service instances to their pods.
+
+The Service Catalog command-line interface (CLI) utility called *`svcat`* is
+available to handle these user related tasks. Although these tasks can be
+handled by various `oc` commands, `svcat`  provides simpler commands to do the
+same tasks, allowing easier interaction with the Service Catalog resources.
```

Comment:
To avoid adverbs and passive voice, can I suggest:

While `oc` commands can perform the same tasks, you can use `svcat` to allow easier interaction with Service Catalog resources.